metrix-simulator/etc/METRIX_En.dic

```diff
@@ -0,0 +1,202 @@
+WARNIncidentGrpPasDePuissance				Warning: No power can be lost during the contingency $1
+WARNLimiteCourantNegative                   Error: Current limit $2 of line $1 is negative $3 MW
+WARNLimiteCourantPlusContraignante          Error: Current limit on $2 ($3) for line $1 is lower than the one for $4 ($5)
+WARNPobjInferieurPmax						Production unit $1: production was set to $2 higher than the maximum power deliverable ($3): Pobj is set to Pmax
+WARNPobjSuperieurPmin						Production unit $1: production was set to $2 lower than the minimum power deliverable ($3): Pobj is set to Pmin
+WARNCuratifNonActiveMaisDef					PST $1 is not in a curative mode , yet some contingencies for curative actions are defined
+WARNTdHs                                    PST $1 is not operational
+WARNAdmitFaible								Warning: admittance for connection $1  is very low( $2)
+WARNAdmitGrande								Warning: admittance for connection $1  is very high ( $2)
+WARNIncCurNonTraiteOuvInc					$2 cannot be used as a curative action on contingency $1 because it makes part of the contingency
+WARNTDCurNonTraite							No curative action for phase shifter $1 because it is not adjustable 
+WARNHVDCCurNonTraiteEntreCompSyn			No curative action possible for HVDC link $1 because it is between 2 different synchronous area
+WARNHVDCCurNonTraite						No curative action for HVDC link $1 because it is not adjustable 
+WARNHVDCConsigneImposee						Control mode of HVDC $1 set to -- Imposed --
+WARNQuadBoucle								Warning: Connection $1 creates a loop on the same node. A breaker on one side had to be opened.
+
+//------------- Curatif topologique --------------------
+ERRCurTopoIncidentInconnu					Outage $1 mentionned in the remedial action list cannot be found
+ERRCurTopoLancementStd						Launch of METRIX without topological remedial actions
+ERRCouplageInconnu							Missing branch for remedial action : '$1'
+ERRCouplageBoucle							Branch '$1' used in a remedial action creates a loop on the same node. This remedial action is not valid.
+ERRCurTopoContrainteInconnue                Constraint '$1' does not match any monitored branch.
+WarnElementDansIncident                     Branch '$1' used in a remedial action is part of the outage '$2'
+WarnElementDejaDansParade                   Duplicate branch '$1' used in a remedial action of outage '$2'
+
+//---------------- Variables li�es ---------------------
+ERRConsoNulle								Error: load value is null on bus '$1' used in variable binding '$2'
+ERRVarRefNulle								Error: reference variable is null for generator '$1' used in variable binding '$2' 
+ERRGroupeLieNonModifiable                   Erreur : generator '$1' used in variable binding '$2' is not modifiable
+ERRConsoLieeNonModifiable                   Erreur : load '$1' used in variable binding '$2' is not modifiable
+WarnMargePetiteGrp							Warning: small margins for generator '$1' used in variable binding '$2'
```

Comment:
To be checked: this file contains 202 lines whereas the french file contains 204 lines.

Reply:
OK